### PR TITLE
Replace logging for large blobs with blob length and hash. 

### LINKED ISSUE #2394: CreateBlob_DisconnectedDuringCreate fails with OOM when run in group
Run all tests

image

### PR DESCRIPTION
**Pull request by karthiknadig**
Fixes #2394

### CODE REVIEW

**Comment by MikhailArkhipov** [comment]
For huge blobs, what is the performance impact when calculating hashes? Just making sure logging is lightweight...

File: src/Host/Protocol/Impl/Message.cs [hunk 1]
```diff
@@ -92,7 +93,14 @@ public class Message {
                 result += $":#{RequestId}#";
             }
 
-            var blobString = BitConverter.ToString(Blob);
+            string blobString = string.Empty;
```
Comment (int19h) [inline src/Host/Protocol/Impl/Message.cs]: This doesn't need the initializer, since it's assigned in both branches below.

**Comment by int19h** [comment]
This is VS-side logging, not broker-side, so I wouldn't anticipate a big impact - it only needs to hash these for one user.

MD5 is very lightweight - around 300-400 megabytes/second on your typical laptop or desktop.

**Comment by int19h** [comment]
👍

**Pull request merged**